### DISCUSSION TITLE
Stern Query Produces No Output: Could it be related to Kubernetes Version?

### DISCUSSION THREAD

**Discussion by helloxjade** [stern/stern#278] [Q&A]
Hello everyone,

I've been facing an issue with querying pod logs using `stern`. Despite using the correct query syntax and namespace, I am not getting any output. I am wondering if this could be related to the version of Kubernetes I am running.

## Environment Details:

- Kubernetes Version: (Please provide the output of `kubectl version`)  v1.19.12
- Stern Version: (Please provide the output of `stern --version`)    1.26.0
- Operating System: (e.g., Ubuntu 20.04, CentOS 8, etc.)  CentOS 7

## Steps to Reproduce:

1. I executed the following command: `stern pod-query -n monitoring`
2. I expected to see logs related to the queried pod(s).
3. What actually happened is that there was no output and no error messages.

## What I Have Tried:

- I have attempted to use different pod queries and namespaces, but the result remains the same.
- I have also checked both `stern` and Kubernetes documentation but found no information related to this issue.

Has anyone encountered a similar issue or has any suggestions to resolve this? Many thanks in advance!

**Comment by tksm** [comment]
Hi,

I verified Stern v1.26.0 worked with Kubernetes v1.19.12 on minikube. Could you try the following command to see if you would see any logs?

```
stern . -n kube-system --tail 10 --since 100000h
```

The `--verbosity=6` option also might help you to troubleshoot.

```
stern . -n kube-system --tail 0 --verbosity 6 --no-follow --only-log-lines
```
